homeassistant/const.py

```diff
@@ -149,6 +149,7 @@
 CONF_WHITE_VALUE = 'white_value'
 CONF_XY = 'xy'
 CONF_ZONE = 'zone'
+CONF_SLOT = 'slot'
```

Comment:
I'm unsure how well this would be accepted, are there other examples for its use, or could this be either defined in the platform or renamed to something else that's already in use?

Reply:
I can't find anything that describes the same thing in const.py, I just had the understanding that they had to be present there to be usable in configurations. I can change it to be local to the platform instead.

Reply:
They are present there to make things consistent between different platforms exposing the same settings, I suppose.